.github/workflows/update-snapcraft.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Create Pull Request
         if: ${{ steps.update_snapcraft.outputs.create_pr == 1 }}
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}  # update this token in repo secret when no longer valid 
+          GITHUB_TOKEN: ${{ github.token }}  # Use the auto-generated action token
```

Comment:
We were using the auto-generated token and changed to personal token because bot-created PRs will not trigger lint, unit, functional tests in their PR and people can accidentally merge them without verifying. But it's unfortunate that the failing error with personal token is silent though, making it hard to know when to update. Maybe we should instead find a way to catch the error instead of reverting the token?

Reply:
here is some information about the behavior you describe: https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#triggering-further-workflow-runs

I will merge this and play with branch protection rules; if I can't find a good way to prevent accidental merges we can rediscuss.

Reply:
Great to know the existence of this branch protection rules. If the PR can be blocked without workflow checks giving +1s, that would be awesome and could solve the issues we've been having in other repos as well.